pegasus/sites.v3/code.org/views/share_privacy.haml

```diff
@@ -15,9 +15,11 @@
                   "Thanks," |
   -# haml-lint:enable MultilinePipe
   - email_subject = "Code.org and your child's privacy"
+  - remind_subject = "Computer Science and Your Child's Privacy:"
```

Comment:
nit: is it worth putting a comment here as to why the email and remind subjects are different?

Reply:
Yes. Committed a new comment